src/lidar_measurement_model_beam.cpp

```diff
@@ -91,6 +96,46 @@ void LidarMeasurementModelBeam::loadConfig(
   int filter_label_max;
   pnh.param("filter_label_max", filter_label_max, static_cast<int>(0xFFFFFFFF));
   filter_label_max_ = filter_label_max;
+
+  pnh.param("add_penalty_short_only_mode", add_penalty_short_only_mode_, true);
+  bool use_raycast_using_dda;
+  pnh.param("use_raycast_using_dda", use_raycast_using_dda, false);
+  if (use_raycast_using_dda)
+  {
+    double ray_angle_half;
+    pnh.param("ray_angle_half", ray_angle_half, 0.25 * M_PI / 180.0);
+    double dda_grid_size;
+    pnh.param("dda_grid_size", dda_grid_size, 0.2);
```

Comment:
I think `dda_grid_size` should be greater than or equal to `max(map_grid_x_, map_grid_y_, map_grid_z_)`. Better to have a validation.

Reply:
Done